### ISSUE TITLE
Changing the TuSimple output format to CULane format

### ISSUE THREAD

**Issue by AirSmoke031** [voldemortX/pytorch-auto-drive#59]
Hello, I know this is not a part of your project, but I hit a deadend with my paper and I cant find the solution.

The papers is about taking 3 different datasets, in my case (Tusimple, CULane and LLAMAS) and using different models to test them, get the output and visualize. Then I have to use the same pretrained model (trained on TuSimple) and test it on the test set of CULane for example and vice versa.

The problem is the output format. LLAMAS (you already adjusted it to CULane format) and CULane use the .txt format so It shouldnt be a problem. However TuSimple has this specific format that they use with the .json file. 

I want to know is there a way, a script, to reformat the .json TuSimple output format to that of CULane.
If I had all the outputs in the same CULane format it would be easier for me to test a model trained on TuSimple and test it on CULane.

Sorry If I am nagging you, your repo is the main building block of my paper.

**Comment by voldemortX** [comment]
@AirSmoke031 If you load a tusimple trained pt file while using a CULane config pipeline (i.e. just modify the model so it can load the tusimple trained pt file).  It should be able to inference. However, I would suggest using the same network arch and max lane num (recommend 4) for simplicity in your case.

**Comment by voldemortX** [comment]
As for label format convertor, we don't have that yet.

**Comment by AirSmoke031** [comment]
Thank you.

**Comment by voldemortX** [comment]
Anyone providing some kind of label conversion script, is encouraged to make a pull request and add them in `/tools/`
